src/backend/marsha/core/serializers/video.py

```diff
@@ -286,6 +286,120 @@ class InitLiveStateSerializer(serializers.Serializer):
     type = serializers.ChoiceField(LIVE_TYPE_CHOICES)
 
 
+class LiveRegistrationSerializer(serializers.ModelSerializer):
+    """Serializer for liveRegistration model."""
+
+    class Meta:  # noqa
+        model = LiveRegistration
+        fields = (
+            "email",
+            "id",
+            "consumer_site",
+            "lti_user_id",
+            "should_send_reminders",
+            "video",
+        )
+        read_only_fields = (
+            "id",
+            "consumer_site",
+            "lti_user_id",
+            "video",
+        )
+
+    # Make sure video UUID is converted to a string during serialization
+    video = serializers.PrimaryKeyRelatedField(
+        read_only=True, pk_field=serializers.CharField()
+    )
+
+    def create(self, validated_data):
+        """Control or set validated_data with token informations.
+
+        Force the video field to the video of the JWT Token if any.
+        Check email, if present in the token, is equal to the one in the request.
+        Set lti informations if they are present in the token.
+
+        Parameters
+        ----------
+        validated_data : dictionary
+            Dictionary of the deserialized values of each field after validation.
+
+        Returns
+        -------
+        dictionary
+            The "validated_data" dictionary is returned after modification.
+
+        """
+        # User here is a video as it comes from the JWT
+        # It is named "user" by convention in the `rest_framework_simplejwt` dependency we use.
+        user = self.context["request"].user
+        video = Video.objects.get(pk=user.id)
+        if video.is_scheduled is False:
+            raise serializers.ValidationError(
```

Comment:
I think DRF expects validation to be made in `validate_*` methods not in the create: https://www.django-rest-framework.org/api-guide/serializers/#validation

The `create` method should be limited to what you have to do to create the object in database.